### ISSUE TITLE
Newer cofhcore required for MFR

### ISSUE THREAD

**Issue by Protheus** [skyboy/MineFactoryReloaded#491]
When I use MineFactoryReloaded-[1.7.10]2.8.1B1-135.jar it tels me to update COFHCore - but there is no new version available. 

Where do I get the latest COFH Core for use with this version of MFR ?

**Comment by skyboy** [comment]
Duplicate of #484

**Comment by Protheus** [comment]
But where isnthe cofhcore used for this version ?

**Comment by Tsyklop** [comment]
where find CoFH 1.7.10 3.1.0 B1?

**Comment by Protheus** [comment]
yes as on curse there is only the 3.0.8 version available and can't seem to find the newer version to run this version of MFR

**Comment by Thiana** [comment]
How is this a duplicate of #484? 484 looks like some sort of incompatibility with GalacticCraft.

The last few months of MFR updates on CF all require a version of CofhCore that doesnt seem to be available to us. Which makes it a little hard to test if an issue we're having is fixed in the newer version before we can report it ;)

**Comment by Protheus** [comment]
Or is the cofh version only for patreon users?

**Comment by Tsyklop** [comment]
@skyboy fix this...
